.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Motivation
+<!-- Why this pull request? -->
+
+### Change description
+<!-- What does your code do? -->
+
+### Other information
+<!-- Any additional information that could help the review process -->
+
+### Reviewer checklist
+
+* [ ] PR address a single concern.
+* [ ] PR title and description are properly filled.
+* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
```

Comment:
```suggestion
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
```
It is more efficient to do things in this order. There is no point in evaluating the quality of title and description if the PR is only to be closed as a duplicate.